src/containers/Tenant/Diagnostics/Healthcheck/Preview/Preview.tsx

```diff
@@ -75,8 +78,8 @@ export const Preview = (props: PreviewProps) => {
     };
 
     return (
-        <DiagnosticCard className={b('preview')}>
-            {renderStatus()}
+        <DiagnosticCard className={b('preview')} active={active}>
```

Comment:
If new view is not activated, Healthcheck should not be shown in the new way.

Reply:
New Healthcheck looks alright with the old design, I don't understand the problem here.

Additionally, we changed not only preview, but also modal content. For the new modal old preview doesn't suits anymore (we show all issues in preview at once, not only selected issue)

Reply:
As for me, if the new design is under construction, the old one should be untouched.
But it's up to the team of course!